### ISSUE TITLE
Pluralization for more languages

### ISSUE THREAD

**Issue by codepunkt** [synacor/preact-i18n#2]
At the moment, only `zero`, `one` and `many` are available. Are there any plans to allow pluralization for languages such as Czech?

See http://www.unicode.org/cldr/charts/29/supplemental/language_plural_rules.html for reference.

**Comment by developit** [comment]
Yes!  They are aliases and you can actually also use an Array or numeric keys to customize per-number.  The thing we don't yet support is custom ranges.  We'd love a PR exploring it!

**Comment by developit** [comment]
FWIW someone mentioned on Twitter that it might be worth investigating ICU Message format, as react-intl uses. For our case this was deemed too large given the set of languages we support, but I'd love to have the ability to inject ICU support via a plugin model.  The definitions are still just strings, but given an ICU plugin they could be extended to be full message templates.

**Comment by yaitskov** [comment]
Russian and Polish languages are also requiring this.
